admin/packaging.py

```diff
@@ -910,6 +910,8 @@ def omnibus_package_builder(
                      flocker_node_path),
                     (FilePath('/opt/flocker/bin/flocker-dataset-agent'),
                      flocker_node_path),
+                    (FilePath('/opt/flocker/bin/flocker-diagnostics'),
+                     flocker_node_path),
```

Comment:
I think I addressed this in a previous pass. Commenting now for completeness & ease of tracking.